README.md

```diff
@@ -84,7 +87,7 @@ Add to your `claude_desktop_config.json`:
   "mcpServers": {
     "my_database": {
       "command": "uv",
-      "args": ["--directory", "/path/to/mcp-alchemy", "run", "server.py"],
+      "args": ["--directory", "/path/to/mcp-alchemy", "run", "mcp_alchemy.py"],
```

Comment:
After publishing to PyPI, the incantation could possibly be compressed to just say:
```shell
uvx mcp-alchemy [--transport=stdio|sse|next]
```